pages/reader/epub.vue

```diff
@@ -471,6 +552,108 @@ function decreaseFontSize() {
   adjustFontSize(-1)
 }
 
+const isShowTextToSpeechOptions = ref(false)
+const isTextToSpeechPlaying = ref(false)
+const isTextToSpeechPaused = ref(false)
```

Comment:
I think "playing" is the opposite of "paused", how about:
```suggestion
const isTextToSpeechOn = ref(false)
const isTextToSpeechPlaying = ref(false)
```